pkgs/applications/version-management/bcompare/default.nix

```diff
@@ -0,0 +1,68 @@
+{ stdenv, qt511, libsForQt511, cairo, gdk_pixbuf, gnome2, bzip2, qt4, lib, fetchurl, autoPatchelfHook, coreutils }:
+
+stdenv.mkDerivation rec {
+  pname = "bcompare";
+  version = "4.2.10.23938";
+
+  src = fetchurl {
+    url = "https://www.scootersoftware.com/${pname}-${version}_amd64.deb";
+    sha256 = "5471d2bcb9494830bc706a940c8a598ecf7a1e56aa7e26c2694048b568d045a0";
```

Comment:
Please convert this to a base32 hash with
```
nix-hash --to-base32 --type sha256 $hash
```

Reply:
done